internal/dnf/dnf.go

```diff
@@ -264,6 +264,14 @@ func (h *historyDB) AddRepoid(ctx context.Context, pkg *claircore.Package) error
 		return nil
 	}
 
+	if ver.Architecture == nil {
+		zlog.Warn(ctx).
+			Err(err).
+			Str("version", v).
+			Msg("unable to parse arch")
+		return nil
+	}
+
```

Comment:
This shouldn't log IMO, and definitely shouldn't log above "debug". Architecture missing is not something that's _always_ unexpected or requires notifying the operator.

Reply:
Kept as `Debug` level in a separate PR: https://github.com/quay/claircore/pull/1674.